ArduPlane/Plane.h

```diff
@@ -439,6 +439,10 @@ class Plane : public AP_Vehicle {
         uint32_t accel_event_ms;
         uint32_t start_time_ms;
         bool waiting_for_rudder_neutral;
+        float throttle_lim_max;
```

Comment:
my concern is these variables may persist longer than we want them to, we could reset them in Plane::update_control_mode() to ensure we can't use stale values